CONTRIBUTING_JS.md

```diff
@@ -132,9 +132,16 @@ Always run tests before pushing and PR'ing your code.
 
 #### Supported Versions
 
-IPFS projects support both the Current and Active LTS versions of Node.js.  Please see [nodejs.org](https://nodejs.org/) for what these currently are.
+The IPFS JavaScript projects follow the most recent LTS of Node.js and respective npm version that gets installed with Node.js. Please consult [nodejs.org](https://nodejs.org/) for LTS timeline. Today these are:
 
-Browser code may be transpiled using Babel to take advantage of the latest JavaScript features.
+```
+» date -u +"%Y-%m-%dT%H:%M:%SZ"
+2018-10-31T09:06:36Z
+» node -v
+v10.13.0
```

Comment:
Are we going to have to update this with every LTS point release? The idea behind saying 'Look at the node website for what the current LTS is' was to reduce the amount of busywork we have to do.

Reply:
I agree, we should not list the version here, it's easy to look up. We should also specify **Active LTS**, as we are not actively supporting Maintenance LTS on our projects.

The language was updated to say `most recent LTS of Node.js`, is the intent to drop node 8 support and only work on node 10, because we should still actively support 8 until it goes to maintenance.